Workiva/dart_dev
# CPLAT-11875: Add Fast Format Command

# [CPLAT-11875](https://jira.atl.workiva.net/browse/CPLAT-11875)
![Issue Status](http://bender.workiva.org:9000/Bender/status/CPLAT-11875)

Added a `hackFastFormat` command to bypass executable generation and just run the formatting command.


## QA'ing
Generally, to run the command globally enable / run the package and run `pub global run dart_dev hackFastFormat {path(s)}`.

Steps:
- Following readme instructions to install the file watcher
- Make sure to globally enable this branch locally
- Update the watcher to execute `pub global run ...` (just need to add the "global" key word)
- Run through the scenarios below
  - Note that it may be beneficial to run the command via cli just to verify the logged command is as expected

Possible scenarios to QA:
- [ ] When the project uses the ORF tool and:
  - [ ] does / doesn't specify a line-length
- [ ] When the project uses the Format tool and:
  - [ ] the project uses the `dartStyle` formatter
  - [ ] the project uses the `dartfmt` formatter
  - [ ] does / doesn't specify command args
- [ ] When either tool specifies a specific file
- [ ] When either tool does not specify a specific file (should fail)
